data/outputs/TEPs/eva71-3C-protease.yaml

```diff
@@ -43,9 +56,129 @@ TEP:
   # List of resources associated with this TEP
   resources:
+    # Construct / plasmid resource
+    - name: A71EV3CPROA-c001
+      # Brief description of the plasmid
+      description:  Construct for enzyme assays crystallography.
+      # Date it was provided
+      date: 2022-10-24
+      # Type: One of ['plasmid', 'xray-fragment-screen', 'pdb-deposition', 'assay', 'compounds']
+      type: plasmid
+      # URL pointing to resource
+      # For a plasmid, this should be an AddGene entry
+      url: N/A 
+      # ASAP Projects and Cores that contributed to this resource
+      # e.g. ['Project 2', 'Structural Biology Core']
+      projects: [Project 2]
+      # Contributors / authors
+      contributors:
+      - Michael Fairhead
+      # Detailed expression protocol associated with this plasmid
+      # This is Markdown: See https://www.markdownguide.org/ for guide    
+      # TODO: In future, this should be a protocols.io link
+      protocol: >      
+        The construct was transformed into the *E. coli* strain BL21(DE3)-RR, and cells were grown at 37°C in TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight by adding 0.5 mM IPTG. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. Protein was lastly purified by SEC (Superose 12 pg,Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis andd mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required. For protein crystallization protein was exchanged into a buffer of 10 mM Bicine pH 8.5, 500 mM NaCl, 5 % Glycerol, 0.5 mM TCEP to allow a higher final protein concentration.
+      # Details associated with this plasmid
+      # TODO: Eliminate this, since this duplicative of the AddGene site.
+      details: |
+        * Vector: pNIC-NHStIIT
+        * Entry clone accession: 
+        * Cell line: 
+        * Tags and additions: His-tag
+        * Construct protein sequence: 
+          MGPSLDFALSLLRRNIRQVQTDQGHFTMLGVRDRLAVLPRHSQPGKTIWVEHKLINILDAVELVDEQGVNLELTLVTLDTNEKFRDITKFIPENISAASDATLVINTEHMPSMFVPVGDVVQYGFLNLSGKPTHRTMMYNFPTKAGQCGGVVTSVGKVIGIHIGGNGRQGFCAGLKRSYFASEQLEHHHHHH
+    - name: A71EV3CPROA-c002 
+      # Brief description of the plasmid
+      description:  Inactive C2A mutant for for enzyme assays.
```

Comment:
@LizbeK : Does the inactive protease really work for enzyme assays? Or do you mean biophysical assays?

Reply:
And does "C2A" mean residue 2 was mutated?

Reply:
The inactive protein was made as negative assay controle for the covalent campaign.

Reply:
Thanks for pointing out terminology error. The active site Cys was mutated to an Ala.